### PR TITLE
Confirm unsaved changes

### PR DESCRIPTION
**Pull request by cedricmessiant**
In any admin section, warn the user if he has unsaved changes when he tries to leave the page
(added flow for the admin pages when they were missing)

Closes DEVAS-1476

### CODE REVIEW

File: assembl/static2/js/app/pages/surveyAdmin.jsx [hunk 1]
```diff
@@ -65,6 +82,7 @@ class SurveyAdmin extends React.Component {
 
       runSerial(mutationsPromises)
         .then(() => {
+          this.setState({ refetching: true });
```
Comment (vincentfretin) [inline assembl/static2/js/app/pages/surveyAdmin.jsx]: Don't you need to set again refatching to false?
`refetchThematics().then(() => this.setState({ refetching: false }));` ?

File: assembl/static2/js/app/router.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useRouterHistory } from 'react-router';
+import { createHistory, useBeforeUnload } from 'history';
+
+import PiwikReactRouter from 'piwik-react-router';
+import get from 'lodash/get';
+import 'bootstrap/dist/css/bootstrap.css';
```
Comment (vincentfretin) [inline assembl/static2/js/app/router.js]: Why do have this bootstrap.css import here? Bad copy paste?

Comment (cedricmessiant) [reply assembl/static2/js/app/router.js]: oh yes, forgot to remove it, thanks

File: assembl/static2/js/app/pages/surveyAdmin.jsx (hunk 1 above)
Comment (cedricmessiant) [reply assembl/static2/js/app/pages/surveyAdmin.jsx]: you're right, it's probably better (even if there is no visible problem as it is)

**Pull request merged**